.github/scripts/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -eux
+
+. .github/scripts/prereq.sh
+. .github/scripts/common.sh
```

Comment:
```suggestion
source .github/scripts/prereq.sh
source .github/scripts/common.sh
```
I thought `.` is "deprecated" or something?

Reply:
No, the dot is the standard: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#dot

`source` is not.